tests/utils.py

```diff
@@ -209,12 +213,9 @@ def hotplug_spec_vm(vm, sockets=None, memory_guest=None):
     ResourceEditor(patches=patch).update()
 
 
-def hotplug_instance_type_vm(vm, sockets=None, memory_guest=None):
-    patch = {
-        vm.vm_instance_type: {"spec": {"cpu": {"guest": sockets} if sockets else {"memory": {"guest": memory_guest}}}}
-    }
+def hotplug_instance_type_vm(vm, new_instance_type_name=None):
```

Comment:
1. Though not new code but should be renamed to indicate what the function does (which is update the resource)
2. `new_instance_type_name` - should be renamed to `instance_type_name` , right? `new` in this context does not necessarily mean anything

Reply:
not addressed